### PR TITLE
[DIST] Initial debianization   

### PR DESCRIPTION
**Pull request by wooksong**
Resolves https://github.com/nnsuite/nnstreamer/issues/1085
See also: https://github.com/nnsuite/nnstreamer/issues/956 

This PR mainly concerns adds files for Debian packaging. To this end, CMake script files are also updated so that the RPM spec is also slightly modified in this PR.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
Nice catch!

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Section: libs
+Priority: optional
+Maintainer: MyungJoo Ham <myungjoo.ham@samsung.com>
+Build-Depends: gcc (>=5), cmake, debhelper (>=9),
+ libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev, libglib2.0-dev,
+ gstreamer1.0-tools, libgtest-dev, ssat, python2.7, nnstreamer-dev,
+ ros-kinetic-catkin, ros-kinetic-cmake-modules,
+ ros-kinetic-genmsg, ros-kinetic-message-generation,
+ ros-kinetic-roscpp
+Standards-Version: 3.9.7
```
Comment (again4you) [inline debian/control:11]: Should we change the version number?

Comment (wooksong) [reply debian/control:11]: @again4you This version is for the debian standards, not for our project.

**Pull request merged**